lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -103,13 +103,13 @@
 
 desc 'Build puppet module package'
 task :build do
-  # This will be deprecated once puppet-module is a face.
   begin
-    Gem::Specification.find_by_name('puppet-module')
-  rescue Gem::LoadError, NoMethodError
-    require 'puppet/face'
-    pmod = Puppet::Face['module', :current]
-    pmod.build('./')
+    require 'pdk/module/build'
```

Comment:
Concerned about having this adding pdk gem as a soft dependency. If the default behavior is to use pdk's build feature as a library method, then perhaps we should add pdk as a runtime dependency.

Reply:
I'm OK with the soft dependency for accessing it as a library but I think there should be a more explicit error message at the end if `pdk` is not available on their PATH, or fall back to loading the old face which I think is what Bryan is suggesting below?